gulpfile.js

```diff
@@ -41,6 +41,7 @@ gulp.task('default', function (callback) {
         'build',
         'rsync',
         'watch',
+        'tdd',
```

Comment:
Mixing PR purpouse with extra stuff? Should be separated to a different PR (probably a slip?)